api/libproject/devstart.rb

```diff
@@ -1120,14 +1121,27 @@ def deploy_gcs_artifacts(cmd_name, args)
 
   common = Common.new
   op = WbOptionsParser.new(cmd_name, args)
+  op.opts.dry_run = false
+  op.add_option(
+    "--dry-run",
+    ->(opts, _) { opts.dry_run = true},
+    "Don't actually push, just log the command lines which would be " +
+    "executed on a real invocation."
+  )
   gcc = GcloudContextV2.new(op)
   op.parse.validate
   gcc.validate
-  common.run_inline %W{gsutil cp scripts/setup_notebook_cluster.sh gs://#{gcc.project}-scripts/setup_notebook_cluster.sh}
+  cmd_prefix = op.opts.dry_run ? DRY_RUN_CMD : []
```

Comment:
I'm wondering if it would be cleaner to have a method like common.run_inline that takes a command and dry_run as a parameter... and just call that everywhere instead of repeating this?

Reply:
Done